README.md

```diff
@@ -20,10 +20,10 @@ do a little setup to get the system working.
 The final goal is to make the hestia platform and thus the server easy to work
 work with, secure and plug and play.
 
-# Run the server
```

Comment:
It will work on any debian based system otherwise they need to use the development setup